### DISCUSSION TITLE
Windows: how to open Preview panel when i open .md file ?

### DISCUSSION THREAD

**Discussion by E834159672** [zed-industries/zed#11665] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

on windows platform , how to open Preview panel when i open .md file 

![屏幕截图 2024-05-10 174020](https://github.com/zed-industries/zed/assets/37103773/235e97ca-e29b-4267-a94f-73c8df9471a6)
![屏幕截图 2024-05-10 174102](https://github.com/zed-industries/zed/assets/37103773/311d3477-6a5c-45a5-964e-6c6ab5488c6b)
![屏幕截图 2024-05-10 174126](https://github.com/zed-industries/zed/assets/37103773/62182641-d94f-4a41-9ab4-4f110b90bfd0)

### Environment

windows 11

**Comment by CharlesChen0823** [comment]
ctrl-shift-p 'toggle markdown preview'

**Comment by E834159672** [comment]
> ctrl-shift-p 'toggle markdown preview'

![1](https://github.com/zed-industries/zed/assets/37103773/69e3c478-ec1e-4d9a-8355-f045eee2eb96)

after i execute this command , nothing happend.

**Comment by E834159672** [comment]
> ctrl-shift-p 'toggle markdown preview'

感谢帮助, 不过我执行了这个命令后,似乎还是没有看到预览, 不知道是不是还有其他的问题? 比如,这个功能是不是需要安装插件?

**Comment by bennetbo** [comment]
Hey @E834159672, you are looking for `markdown: Open preview` not `pane: toggle preview tab`.


<img width="601" alt="image" src="https://github.com/zed-industries/zed/assets/53836821/d3551a01-e348-4977-86df-041c00cbe3ef">

**Comment by E834159672** [comment]
> Hey @E834159672, you are looking for `markdown: Open preview` not `pane: toggle preview tab`.
> <img alt="image" width="601" src="https://private-user-images.githubusercontent.com/53836821/329549729-d3551a01-e348-4977-86df-041c00cbe3ef.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUzNDI0NzMsIm5iZiI6MTcxNTM0MjE3MywicGF0aCI6Ii81MzgzNjgyMS8zMjk1NDk3MjktZDM1NTFhMDEtZTM0OC00OTc3LTg2ZGYtMDQxYzAwY2JlM2VmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEwVDExNTYxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkwMmQyNzgyMTgxYmZmMjhlYjAzNWZkYWE1YmIzMGE0OGFkNGY4YzUwZjExYjkyMmY4YjYyMTI0Mjc3ODdkZTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ppg78pIcczae8IbwRK-jgcY09AqKegjKPw-fxadMy34">

 following your method ,I finally found it, thank you.

![2](https://github.com/zed-industries/zed/assets/37103773/486f54e1-137e-4e51-b86d-9d52bbec378b)


i suggest add a context menu.
